js/renovation/button.tsx

```diff
@@ -242,24 +219,36 @@ export default class Button extends JSXComponent(ButtonProps) {
   }
 
   get aria() {
-    return getAriaLabel(this.props.text, this.props.icon);
+    const { text, icon } = this.props;
```

Comment:
Is there any reason why we get rid of `getAriaLabel` function?

Reply:
This function is small and used only in aria getter, so why do we need it at all? All that stuff is just getter